README.md

```diff
@@ -46,7 +46,7 @@ config in init, adjust accordingly):
 
```

Comment:
Dated in what sense? If it's not correct, it should go but otherwise, it might be a good idea to reorganise the docs so we have cheat sheet-esque examples for getting started.

Reply:
Only dated in that there are newer Inky models that people might be more likely to have. Since this PR is for the Impression, I was thinking it would be nice to see an example for how to use one. Of course, more examples are always useful and it would be nice to have one for each model, but that seems like it's outside of the scope of this PR.

Reply:
In that case I'd just say to add structure (i.e. minimal work to create paragraphs) and introduce the example for the newer hardware. If it gets to busy or whatever, someone who feels passionately about documentation would be able to swing by and put that stuff somewhere else and augment it as they see fit.

Just feels like it'd be a shame to throw away correct documentation.

Reply:
Oh, and just to be clear, I completely agree that a simple usage example would be good to have!

Reply:
I've updated the readme in c81122d5fbd10f22a6643bf534a7ed3360002139 although I didn't get around to adding a impression example, although I agree that one would be nice! Maybe a pure-inky version of the color test that shows all the colors.